test/media-segment-request.test.js

```diff
@@ -256,7 +256,7 @@ QUnit.test('aac with id3 will make it to the transmuxer', function(assert) {
   this.options.doneFn = () => {
     assert.deepEqual(this.calls, {
       data: 1,
-      trackInfo: 48,
+      trackInfo: 1,
```

Comment:
We now only fire trackinfo on the muxer once for audio, rather than every time we fire data. The partial muxer and video stream already did this.